components/__utils__/mountWithStore.ts

```diff
@@ -1,48 +1,47 @@
-// @flow
 import assert from 'assert'
 import * as React from 'react'
 import { Provider } from 'react-redux'
 import { mount } from 'enzyme'
 
 import type { ReactWrapper } from 'enzyme'
 
-export type MockStore<State, Action> = {|
-  getState: JestMockFn<[], State>,
-  subscribe: JestMockFn<[], void>,
-  dispatch: JestMockFn<[Action], Action>,
-|}
-
-export type WrapperWithStore<Element, State, Action> = {|
-  wrapper: ReactWrapper<Element>,
-  store: MockStore<State, Action>,
-  refresh: (nextState?: State) => void,
-|}
-
-export type MountWithStoreOptions<State> = {
-  initialState?: State,
-  ...
+export interface MockStore<State, Action> {
+  getState: jest.MockedFunction<() => State>
+  subscribe: jest.MockedFunction<(args: []) => void>
+  dispatch: jest.MockedFunction<(...actions: Action[]) => Action>
```

Comment:
The original flow type is a tuple representing a single action param, not an array of actions

```suggestion
  dispatch: jest.MockedFunction<(action: Action) => Action>
```